DecipherNow/gm-fabric-dashboard
# fix: replace Nunito with NunitoSans

Resolves #1343 

## Proposed Changes

  - Replace Nunito with NunitoSans
  - Remove NunitoSans from the project
  
## How to Test

  - Start the app
  - Use WhatFont or Chrome Dev Tools to confirm that NunitoSans is rendering as the default font
